def test_kasitli_hata():
      assert True, 'Bu test kasıtlı olarak başarısız!'